### PR TITLE
Installs react facebook login, implements login with facebook into lo…

### PR DESCRIPTION
**Pull request by michelleann815**
Completes ticket: 

https://trello.com/c/OlwXM8fa
or
https://trello.com/c/OlwXM8fa/28-add-option-for-sign-in-with-facebook

Tasks completed:
-Create an OAuth2 client with Facebook
-Get clientID from Facebook
-install and run npm react-facebook-login
-import FacebookLogin component into login component
-use the FacebookLogin code from the npm package doc

### CODE REVIEW

File: client/src/Components/Login.js [hunk 1]
```diff
@@ -76,6 +77,10 @@ const Login = () => {
     return <Redirect to="/" />;
   }
 
+  const responseFacebook = response => {
+    console.log(response);
```
Comment (reneefromhold) [inline client/src/Components/Login.js:81]: Remove this if there is no use outside of console/debug

File: client/src/Components/Login.js [hunk 2]
```diff
@@ -132,6 +137,16 @@ const Login = () => {
             Continue with Google
           </h4>
         </div>
+        <div className={classes.loginService}>
+          <FacebookLogin
+            appId="534803271003174"
```
Comment (reneefromhold) [inline client/src/Components/Login.js:142]: Is this specific to your account? If yes, let's make this config based in the .env file

**Review by reneefromhold — CHANGES_REQUESTED** [review]
Let's talk about the appId for the Facebook login. This is something I'm unfamiliar with.
This value should be .env file based. Please create a document on what other devs onboarding to perch will need to set up to work with this feature.

**Comment by michelleann815** [comment]
Can no longer use Facebook login:
https://developers.facebook.com/blog/post/2018/06/08/enforce-https-facebook-login/
https://stackoverflow.com/questions/66848225/facebook-app-localhost-force-https-problem

**Pull request closed without merging**